MetadataMigration/src/main/java/org/opensearch/migrations/commands/Evaluate.java

```diff
@@ -38,10 +37,9 @@ public EvaluateResult execute(RootMetadataMigrationContext context) {
                 .build();
         } catch (Throwable e) {
```

Comment:
This should most definitely NOT be throwable.  Unless you have very good reason to handle Errors, don't catch Throwables.  I've only know of one place where we need to catch one (while shutting down the replayer, which can uniquely die from > 1 signal/error).

Reply:
- Resolving this out of band in https://github.com/opensearch-project/opensearch-migrations/pull/1631